storage/coop/rchain/key-value-store/KeyLexer.scala

```diff
@@ -95,30 +95,24 @@ class KeyLexer(lineIn: String) {
     var endOfString = false
 
     while (i < line.length && c != '(' && c != ')' && c != ',') {
-      c match {
-        case _ if Character.isLetter(c) => {
-          if (isDigits)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (4): '"
-                + line + "', " + i)
-          isLetters = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ if Character.isDigit(c) => {
-          if (isLetters)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (5): '"
-                + line + "', " + i)
-          isDigits = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ =>
+      if (Character.isLetter(c)) {
+        if (isDigits)
           throw new Exception(
-            "KeyLexer.NextToken(): malformed (6): '"
+            "KeyLexer.NextToken(): malformed (4): '"
               + line + "', " + i)
-      }
+        isLetters = true
```

Comment:
Not a change in this PR, but does this parser not allow letter-digit, _e.g._, `a0` as a word?

Reply:
a0 is not legal as it stands.